card/card.gemspec

```diff
@@ -28,24 +28,9 @@ Gem::Specification.new do |s|
   s.homepage      = "http://decko.org"
   s.licenses      = ["GPL-2.0", "GPL-3.0"]
 
-  s.files         = `git ls-files`.split $INPUT_RECORD_SEPARATOR
-
-  # add submodule files (seed data)
-  morepaths = `git submodule --quiet foreach pwd`.split $OUTPUT_RECORD_SEPARATOR
-  morepaths.each do |submod_path|
-    gem_root = File.expand_path File.dirname(__FILE__)
-    relative_submod_path = submod_path.gsub "#{gem_root}/", ""
-    Dir.chdir(submod_path) do
-      morefiles = `git ls-files`.split /\n/ #$OUTPUT_RECORD_SEPARATOR
-      #puts "ors = #{$OUTPUT_RECORD_SEPARATOR}"
-      s.files += morefiles.map do |filename|
-        "#{relative_submod_path}/#{filename}" #unless filename =~ /\s/
-      end.compact
-    end
-  end
-
+  s.files         = Dir["VERSION", "README.rdoc", "LICENSE", "GPL",
+                        "{config,db,lib,mod,tmpsets}/**/*"]
 
-  s.test_files    = s.files.grep(%r{^(test|spec|features)/})
```

Comment:
any reason not to have `s.test_files`?

Reply:
It seems to be deprecated and I don't see the value of adding test files to the gem